S7.Net/Types/Date.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using S7.Net.Helper;
+
+namespace S7.Net.Types
+{
+    /// <summary>
+    /// Contains the conversion methods to convert Words from S7 plc to C#.
+    /// </summary>
+    public static class Date
+    {
+        /// <summary>
+        /// Minimum allowed date for the IEC date type
+        /// </summary>
+        public static readonly System.DateTime IecMinDate = new(year: 1990, month: 01, day: 01);
```

Comment:
```suggestion
        public static System.DateTime IecMinDate { get; } = new(year: 1990, month: 01, day: 01);
```
To expose it as property instead of field?

Same below for `IecMaxDate`.